packages/showcases/css/stories/components/navigation/tabs/examples/demo.html

```diff
@@ -1,35 +1,31 @@
 <div class="block">
-  <ul
+  <div
     id="demo-tabs-tablist"
     class="vtmn-tabs"
     role="tablist"
     aria-label="Tabs navigation"
+    tabindex="-1"
```

Comment:
As far as I know, `tabindex="-1"` on a non-focusable element make it only focusable by a script. Why is it necessary here ?